packages/@okta/vuepress-theme-prose/global-components/StackSelector.vue

```diff
@@ -50,24 +51,30 @@
         if (value && value.link) {
           this.hasFocus = true;
           this.$router.push(value.link);
+
+          // After new value selected we record new route value and prev value into shared stackSelectorData object
+          // to be able to use it in SidebarItem.vue component
+          this.stackSelectorData.from = this.selectedOption.link;
+          this.stackSelectorData.to = value.link;
```

Comment:
can you mutate `stackSelectorData` in top-level component?
(https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/layouts/Layout.vue#L107)